simulators/mujoco_simulator/revolve2/simulators/mujoco_simulator/textures/_checker.py

```diff
@@ -0,0 +1,17 @@
+from dataclasses import dataclass, field
+
+from revolve2.simulation.scene import Color
+from revolve2.simulation.scene.geometry.textures import Texture, TextureReference
+
+from ._mujoco_builtin_references import MujocoBuiltinReferences
+
+
+@dataclass(kw_only=True, frozen=True)
+class Checker(Texture):
+    """A checker texture for geometric models."""
+
+    reference: TextureReference = field(default=MujocoBuiltinReferences.CHECKER.value)
```

Comment:
I think the way it's implemented now it would be good enough to directly write "checker" here and leave out the enum :) it's already properly seperated for the user through the different classes :v: